eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -1,22 +1,44 @@
 parameters:
+  DisplayName: ''
```

Comment:
Is this needed? I don't think it is based on the current implementation.

Reply:
Removed.